public/app/plugins/datasource/kairosdb/queryCtrl.js

```diff
@@ -53,7 +53,7 @@ function (angular, _) {
         return metricList;
       }
       else {
-        $scope.datasource.performMetricSuggestQuery().then(function(result) {
+        $scope.datasource._performMetricSuggestQuery().then(function(result) {
```

Comment:
this looks wrong, the controller should not access a private function (the leading underscore), should this not be metricFindQuery('metrics()')